lib/libnotify/notify_client.c

```diff
@@ -50,7 +50,7 @@
 #endif
 #include <Block.h>
 #include <dispatch/dispatch.h>
-#include <dispatch/private.h>
+// #include <dispatch/private.h>
```

Comment:
Why was this removed - not needed? TBH I'm not sure why it is included in the first place outside of libdispatch.

Reply:
Didn't seem to be needed, also there is no `dispatch/private.h`, in the source it's `./lib/libdispatch/private/private.h` so I'm not sure how it was working anywhere else tbh.